ajax/libs/vue-tables-2/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "vue-tables-2",
+  "filename": "vue-tables-2.min.js",
+  "description": "Vue.js 2 grid components",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matfish2/vue-tables-2"
+  },
+  "keywords": [
+    "vue",
+    "vuex",
+    "bootstrap",
+    "table",
+    "grid"
+  ],
+  "author": "Matanya Fishaimer <matfish2@gmail.com>",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/matfish2/vue-tables-2.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.+(js)"
```

Comment:
This might be my lack of familiarity with glob patterns, but is this combining one with a regex? Is it the best way to match any js extension?

Reply:
TBH, I copied the file over from another repo and made the necessary adjustments, so if you know a better way to match those files I can change accordingly

Reply:
Looking at the files in the repo (https://github.com/matfish2/vue-tables-2/tree/v2.0.0-beta.1/dist), a simple glob of `*.js` would suffice here. `**/*.js` if you expect to be using sub-dirs in `dist` anytime soon.